article/app/renderers/RemoteRenderer.scala

```diff
@@ -46,6 +50,9 @@ class RemoteRenderer extends Logging {
             case 200 =>
               Cached(article)(RevalidatableResult.Ok(Html(response.body)))
                 .withHeaders("X-GU-Dotcomponents" -> "true")
+            case 404 =>
+              Cached(CacheTime.NotFound)(WithoutRevalidationResult(NotFound))
```

Comment:
I saw somewhere else earlier where 404 wasn't cached (after CAPI returns 404); thinking about it I suppose it doesn't make it back directly to fastly from here to be cached anyway. Not sure whether it'd be clearer to cache or not in this case.